Task_HelloWorld/index.html

```diff
@@ -1,15 +1,24 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Hello World Problem</title>
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <div top="0">
```

Comment:
The 'top' attribute is not a valid attribute for a div element. Consider removing it or using a different attribute.